docs/src/main/sphinx/connector/hive.rst

```diff
@@ -145,30 +145,23 @@ different behaviors to the SQL version, the returned results may differ. In more
 extreme cases the queries might fail, or not even be able to be parsed and
 executed.
 
-**Experimental**
+**Advanced**
```

Comment:
@claudiusli and @joshthoward  .. how was the decision made to not call it experimental anymore? Did Coral mature enough now and we got confirmation about that with LinkedIn? Did development stabilize and slow down in Trino?